library_generation/test/resources/google-cloud-java_config.yaml

```diff
@@ -0,0 +1,52 @@
+#Required.
+gapic_generator_version: 2.30.0
+#Required.
+googleapis-commitish: 4512234113a18c1fda1fb0d0ceac8f4b4efe9801
+#Required.
+owlbot-cli-image: sha256:623647ee79ac605858d09e60c1382a716c125fb776f69301b72de1cd35d49409
+#Required.
+synthtool-commitish: 59fe44fde9866a26e7ee4e4450fd79f67f8cf599
+#Required.
+python-version: 3.11.2
+#Optional. The root folder name of generated client libraries. If empty, modules will be created under current folder, useful for single module
+destination-path: google-cloud-java
+#Optional. This is a relative path to destination-path above. We will create a new versions.txt file if it is not specified
+versions_path: versions.txt
+#Required.
+products:
+    #Required. Can be used for populating the folder name java-{api_shortName}. This is also the destination-name in new-client.py.
+  - api_shortname: asset
```

Comment:
Could you write an example of java-bigtable, which combines two clients: bigtable and bigtable-admin?